trio_async/starter.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+import trio_asyncio
+from temporalio.client import Client
+
+from trio_async import workflows
+
+
+@trio_asyncio.aio_as_trio  # Note this decorator which allows asyncio primitives
```

Comment:
Could you expand this comment slightly, are we basically saying

> without this decorator you wouldn't be able to use anything from the `asyncio` module.

?

Reply:
Yes, "allows asyncio primitives" was meant to imply "needed to allow asyncio primitives", otherwise if it wasn't needed we could remove the decorator. I can rephrase if needed (though don't want to effectively re-document https://trio-asyncio.readthedocs.io/en/latest/usage.html#calling-asyncio-from-trio).